Common/Basic comppenants/Filters.cs

```diff
@@ -0,0 +1,42 @@
+﻿using OpenQA.Selenium;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Infrastructure
+{
+    public class Filters : ComppenantBase
+    {
+        public List<Button> ColorFilter => ParentElement.FindElements(By.CssSelector("#ul_layered_id_attribute_group_3 li")).Select(element => new Button(Driver, element)).ToList();
+        public IWebElement PriceRange => ParentElement.FindElement(By.CssSelector("#layered_price_range"));
+
+
+        public Filters(IWebDriver driver, IWebElement parentElement) : base(driver, parentElement)
+        {
+
+        }
+
+        public double[] GerPriceRange()
+        {
+            var priceRangeString = PriceRange.ToString();
```

Comment:
you do a ToString to an IWebElement. Check if this works - you need to get the value of the element